code/src/nuvla/connector/kubernetes.py

```diff
@@ -441,6 +473,30 @@ def _get_component_logs(self,
          'StatefulSet',
          'DaemonSet']
 
+    def sanity_namespace(self, namespace):
+        '''
+        Check that the namespaces makes sense
+        A kubernetes NE will run in a namespace of type nuvla{edge|box}-UUID
+        whereas a deployment will run in a namespace type UUID only
+        '''
+
+        output = self._exec_stdout_json(['get', 'namespaces',
+                                       '-o', 'json'])
+        namespaces = output.get('items',[])
+        for nns in namespaces:
+            ns = nns['metadata']['name']
+            log.debug(f"checking namespace: {ns}")
+            if namespace == ns:
+                return namespace
+            if namespace in ns:
+                log.info(f"The namespace: {namespace} has been corrected to: {ns}")
+                return ns
+
+        # we should throw an exception here? FIXME
```

Comment:
The sanity_namespace is not used anymore. For the moment.
If we move to get the deployment logs to use this class then it could be useful again.
Keep in the code for now?